src/Features/VisualBasic/Portable/Completion/CompletionProviders/EnumCompletionProvider.vb

```diff
@@ -33,7 +33,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
                                                                    Return m.Kind = SymbolKind.Field AndAlso
                                                                       DirectCast(m, IFieldSymbol).IsConst AndAlso
                                                                       m.IsEditorBrowsable(hideAdvancedMembers, context.SemanticModel.Compilation)
-                                                               End Function))
+                                                               End Function).Concat(enumType))
```

Comment:
comment this.  it's also very hard to notice that piece of code.  might want to break it up into two statements.